assets/src/date/getTimeFromNow.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import moment from 'moment-timezone';
+
+/**
+ * Internal dependencies
+ */
+import { getDateObjectWithTimezone } from './getDateObjectWithTimezone';
+
+/**
+ * @description Get the time passed from or time until a date to the time now in any timezone
+ * @param {Date} date Uses moment to find time passed since/until.
+ * If date is not an instance of moment when passed in it will create a moment from it.
+ * @param {Object} dateFormatting Object responsible for relevant date formatting.
+ * Should contain dateFormat, timezone, gmtOffset, and timeFormat - all strings.
+ * @return {string} Displayable relative date string
+ */
```

Comment:
```suggestion
/**
 * Get the time passed from or time until a date to the time now in any timezone.
 *
 * @param {Date} date Uses moment to find time passed since/until.
 * If date is not an instance of moment when passed in it will create a moment from it.
 * @param {Object} dateFormatting Object responsible for relevant date formatting.
 * Should contain dateFormat, timezone, gmtOffset, and timeFormat - all strings.
 * @return {string} Displayable relative date string
 */
```

If you describe a symbol at the very beginning of a JSDoc comment, before using any block tags, you may omit the `@description` tag.